recipe/build.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+set -euo pipefail
+
+cd src
+# Remove this stuff so that lib is not polluted
+rm -Rf .fslckout ChangeLog tests demo BWman
+[[ ! -d $PREFIX/lib/tcl8.6/bwidget ]] || exit 1
+mkdir -p $PREFIX/lib/tcl8.6/bwidget
+cp -r * $PREFIX/lib/tcl8.6/bwidget/
```

Comment:
will not be better to be on par how we populate the bwidget folder for both unix & windows ?